collections/_infographics/policies/emission-pledges.md

```diff
@@ -12,4 +12,34 @@ data-orig:
   - [ "Our Jupyter notebook", "https://github.com/faktaoklimatu/data-analysis/blob/master/notebooks/net-zero-pledges.ipynb" ]
 ---
 
-{% include includes-local/comment-placeholder.md slug="infografiky/emisni-zavazky" %}
+## How to read an infographic?
+{% comment %}
+{% include preview-box.html
+    title="What is carbon neutrality?"
+    text="We explain exactly what the zero carbon commitment means and how the term can be confusing."
+    slug="uhlikova-neutralita"
+%}
+{% endcomment %}
+This infographic shows the share of carbon-neutral countries in global CO<sub>2</sub> emissions in 2018. Excluding the Land Use, Land Use Change and Forestry sector (*LULUCF), all CO<sub>2</sub> emissions, including international maritime and aviation transport. **As of March 2022, 87% of global emissions come from countries moving towards carbon neutrality**, with 131 countries total.
+
+Countries that are moving towards carbon neutrality are those that have commitments: fulfilled (e.g., Bhutan), set in law, set in a policy document, in the form of a declaration/commitment, or at the proposal/discussion stage. If we exclude countries with commitments in the draft or discussion phase, there would be a 3% reduction in covered emissions to 84% of global emissions.
+
+Some countries claim to be climate neutral (*climate neutrality, net-zero emissions*) and some to carbon neutrality (*carbon neutrality, net-zero carbon*). However, we do not distinguish between them in the infographic and use the term carbon neutrality for all commitments. While the former refers to all greenhouse gases, the latter refers "only" to CO<sub>2</sub>. I.e., if a country commits to climate neutrality, this commitment includes carbon neutrality. <!--For more information on carbon neutrality in general, see our [explainer](/explainers/carbon-neutrality).--> Countries worldwide must achieve carbon neutrality by around 2050 to keep global warming below 1.5°C. Commitments towards later dates are insufficient to meet this goal.
+
+## More details on individual countries
+
+Of the countries that account for more than 1% of global emissions, only Iran and Mexico have not signed up for carbon neutrality.
+The five largest emitters are the following countries/groups, which together account for 63.2% of global CO<sub>2</sub> emissions:
+
+1. **China**: 29.9% share of global CO<sub>2</sub> emissions, carbon neutrality deadline is 2060,
+2. **USA**: 13.7% share, 2050,
+3. **EU-27**, 8.2% share, 2050
+4. **India**, 6.7% share, 2070,
+5. **Russia**, 4.8% share, 2060.
+<!--
+We summarise the commitments of other states in the research paper [Emission commitments of states](/studie/2021-reserse-zavazky-statu).
+-->
+## Sources
+
+* The source of carbon neutrality terms is the [Net Zero Tracker](https://zerotracker.net) project.
```

Comment:
I would change "Sources" to "References"; more standard.

Reply:
change "The source of carbon neutrality terms is the [Net Zero Tracker](https://zerotracker.net) project." to "The reference for the carbon neutrality terms is the [Net Zero Tracker](https://zerotracker.net) project."

Reply:
I updated the word "Source" and text to use "References".